### ISSUE TITLE
[HOLD for payment 2023-09-04] View distance details in App

### ISSUE THREAD

**Issue by neil-marcellini** [Expensify/App#22713]
- HELD on https://github.com/Expensify/App/issues/22712

Please follow the plan in the design doc: View [distance details in App](https://docs.google.com/document/d/16Q9sE-991CZ4wUcalOs_U-qyEEza_C9tZwj6G5lHUp0/edit#heading=h.k49t5tr95nym)

**Comment by arosiclair** [comment]
Still held

**Comment by luacmartins** [comment]
Still on hold

**Comment by arosiclair** [comment]
I'll start progress on this

**Comment by arosiclair** [comment]
No real progress atm. This has hard dependencies for helper functions coming from https://github.com/Expensify/App/issues/22710. Good thing is that it should be easy to move on once that issue is done

**Comment by arosiclair** [comment]
Posted a draft PR. Not as many dependencies as I thought because we actually don't have to recompute the merchant text ("171 miles @ $0.55 / mile") since it is already saved on the transaction. 

Still need to make a slight change to display the route image as it won't be saved exactly the same as a normal receipt (something we might want to change later)

**Comment by arosiclair** [comment]
Implemented and reviewed internally so no payment. Closing
